app/api/views.py

```diff
@@ -140,7 +140,16 @@ class DocumentList(generics.ListCreateAPIView):
     def get_queryset(self):
         project = get_object_or_404(Project, pk=self.kwargs['project_id'])
 
-        queryset = project.documents
+        if (settings.ELASTIC_SEARCH_HOST):  # Elasticsearch
+            from .index import DocumentIndex
```

Comment:
I know this import shouldn't go here ... but anyone have recommendations on conditional imports? I only want this Index to be loaded in if we are using Elasticsearch.